x-pack/plugins/ml/server/shared_services/providers/trained_models.ts

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type * as estypes from '@elastic/elasticsearch/lib/api/typesWithBodyKey';
+import { KibanaRequest, SavedObjectsClientContract } from '@kbn/core/server';
+import { GetGuards } from '../shared_services';
```

Comment:
```suggestion
import type { KibanaRequest, SavedObjectsClientContract } from '@kbn/core/server';
import type { GetGuards } from '../shared_services';
```

Reply:
Updated in https://github.com/elastic/kibana/pull/141274/commits/6b418e63ca7d830bea9e3f55a0a5d83512a899f7